plugins/MonitorStage/MonitorMain.qml

```diff
@@ -12,7 +12,7 @@ Rectangle
     id: viewportOverlay
 
     property bool isConnected: Cura.MachineManager.activeMachineHasNetworkConnection || Cura.MachineManager.activeMachineHasCloudConnection
-    property bool isAbstractCloudPrinter: Cura.MachineManager.activeMachineIsAbstract // && Cura.MachineManager.activeMachineHasCloudRegistration
+    property bool isAbstractCloudPrinter: Cura.MachineManager.activeMachineIsAbstract
```

Comment:
Oops, I don't think that should have been commented out in the first place. I think it now also may send 3rd party abstract printer users to our Digital Factory? 😅

Reply:
Yeah, that's also why this shouldn't live here. The output device that is active for abstract machines should handle this. I'm currently implementing that as part of CURA-8463